presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -153,6 +153,23 @@ Mathematical Functions
     array ``bins``. The ``bins`` parameter must be an array of doubles and is
     assumed to be in sorted ascending order.
 
+Statistical Functions
+-----------------------
+
+.. function:: wilson_interval_lower(pos, neg[, z]) -> double
```

Comment:
Yes, if it's constant, the optimizer will constant fold it. You can actually see that in the explain plan.